### ISSUE TITLE
ERROR: Value provided for user name, '_' (type str), is too short when running conan remote clean

### ISSUE THREAD

**Issue by ssrobins** [conan-io/conan#5606]
1. `git clone https://gitlab.com/ssrobins/sdl2-example.git`
2. `cd sdl2-example`
3. Run the `build_` script matching your platform
4. `conan remote clean`

I get this error:
```
ERROR: Value provided for user name, '_' (type str), is too short. Valid names must contain at least 2 characters.
```

If I run `conan remove "*" -f` then `conan remote clean`, no error. I'm using the new experimental feature that allows me to do without the user and channel so I think it's having a problem with the single `_` that replaces the user in the package path. Not a big deal, but thought you'd want to know.

Environment:
Conan 1.18.1
macOS 10.14.4/Windows 10 Pro 64-bit 1809

Hope to see you at CppCon so I can personally thank you all for a great job with Conan!

**Comment by memsharded** [comment]
Hi @ssrobins 

Thanks very much for testing this new feature and reporting it. It is certainly a bug, I have submitted a fix in https://github.com/conan-io/conan/pull/5607, and planned to release it in a minor 1.18.2.

Yes! We will be at CppCon, that is great, lets meet and talk there!

**Comment by memsharded** [comment]
Fixed by https://github.com/conan-io/conan/pull/5607, will be released in 1.18.2

**Comment by keef-cognitiv** [comment]
I end up having this same problem while running a conan install.
```
ERROR: Failed requirement 'zlib/1.2.8' from 'aws-sdk-cpp/1.7.212@kmaragon/stable'
ERROR: '_' is too short. Valid names must contain at least 2 characters.. [Remote: cognitiv]
```
Conan version 1.19.1
